politeiad/politeiad.go

```diff
@@ -331,6 +331,13 @@ func (p *politeia) newRecord(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
+		if err == backend.ErrQuiesced {
+			log.Errorf("%v politeiad is quiesced, writes aren't allowed",
```

Comment:
There's no need to log this.  You're already returning an error with the same information in it.  This is redundant and `log.Error` should only be used for unexpected errors.